### PR TITLE
Megh/swervewheelmethods

### PR DESCRIPTION
**Pull request by Meghd2cx**
Added swerve helper methods : MPStoVoltOutput and RangeFix

### CODE REVIEW

File: code/src/main/java/frc/robot/subsystems/SwerveWheel.java [hunk 1]
```diff
@@ -124,11 +134,76 @@ public void setDesiredState(SwerveModuleState state) {
...
+        m_turningEncoder.getRadians(), (rangeFix(state.angle.getRadians()))));
     // Calculate the turning motor output from the turning PID controller.
     m_driveMotor.set(driveOutput);
     m_turningMotor.set(turnOutput);
   }
+
+  /**
+   *
+   *
+   * @param state Desired state with speed and angle
```
Comment (dotnetprofessional) [inline code/src/main/java/frc/robot/subsystems/SwerveWheel.java:148]: clean up comments

File: code/src/main/java/frc/robot/subsystems/SwerveWheel.java [hunk 2]
```diff
@@ -124,11 +134,76 @@ public void setDesiredState(SwerveModuleState state) {
...
+   * @return scaled voltage needed to attain specified speed
+   */
+  
+  public double mpsToVoltOutput (double WheelMps){
+
+    double secToMin = 60.0;
+    double WheelMpm = WheelMps*secToMin;
+
+    double WheelDiameter = _config.SwerveWheel.WheelDiameter;
+    double WheelCircum = WheelDiameter*Math.PI;
```
Comment (dotnetprofessional) [inline code/src/main/java/frc/robot/subsystems/SwerveWheel.java:167]: unused

File: code/src/main/java/frc/robot/subsystems/SwerveWheel.java [hunk 3]
```diff
@@ -124,11 +134,76 @@ public void setDesiredState(SwerveModuleState state) {
...
+   *
+   *
+   * @param WheelMPs given speed of the wheel in MPS
+   * @return scaled voltage needed to attain specified speed
+   */
+  
+  public double mpsToVoltOutput (double WheelMps){
+
+    double secToMin = 60.0;
+    double WheelMpm = WheelMps*secToMin;
```
Comment (dotnetprofessional) [inline code/src/main/java/frc/robot/subsystems/SwerveWheel.java:164]: local variables should use camel case, actually in Java everything except classes uses camel case.

**Comment by noam987** [comment]
Didn't work, we can merge after we debug it.

**Pull request closed without merging**